sycl/source/detail/global_handler.cpp

```diff
@@ -355,33 +355,39 @@ void shutdown_late() {
 extern "C" __SYCL_EXPORT BOOL WINAPI DllMain(HINSTANCE hinstDLL,
                                              DWORD fdwReason,
                                              LPVOID lpReserved) {
-  bool PrintUrTrace =
-      sycl::detail::ur::trace(sycl::detail::ur::TraceLevel::TRACE_CALLS);
+  try {
```

Comment:
Need to minimize try-catch block entirely to only calls which throw exceptions. Is it the only one - `shutdown_win()`?

Reply:
No, sycl::detail::ur::trace() throws an exception. Actually the origin is far down its calling chain.

Reply:
OK, in that case only `sycl::detail::ur::trace(...)`  call should be under try-catch block.

Reply:
Sure. Done.